kibana/config/kibana.yml

```diff
@@ -0,0 +1,77 @@
+# Kibana is served by a back end server. This controls which port to use.
+port: 5601
+
+# The host to bind the server to.
+host: "0.0.0.0"
+
+# The Elasticsearch instance to use for all your queries.
+elasticsearch_url: "http://192.168.1.112:9200"
```

Comment:
This has to be externalized. When ever there is an ip change, you shouldn't expect people to rebuild the image. Move the elastic search url when you are starting the container and pass it as a run time parameter in docker run